resources/dicts/patrols/general/border.json

```diff
@@ -3001,5 +3003,475 @@
+        "cats_from": ["n_c:0"],
+        "mutual": true,
+        "values": ["trust", "comfort"],
+        "amount": 28
+},	
+{
+        "cats_to": ["p_l"],
+        "cats_from": [ "clan" ],
+        "mutual": false,
+        "values": ["trust"],
+        "amount": -15
+}
+],
+"new_cat": [
+    ["clancat", "mate:p_l", "age:mate"]
+],
+"art": "gen_flower_date_warrior_other_clan"
+},
+{
+    "text": "n_c:0 urges p_l to give up {PRONOUN/p_l/poss} life in c_n and join {PRONOUN/n_c:0/object} in o_c_n. When p_l refuses, n_c:0 bows {PRONOUN/n_c:0/poss} head and admits {PRONOUN/n_c:0/subject} can't live without {PRONOUN/p_l/object}. n_c:0 will wait for p_l beyond the border, until p_l is ready to start their life together away from the Clans.",
+    "exp": 20,
+    "weight": 30,
+    "other_clan_rep": -1,
+    "relationships": [
+{
+            "cats_to": ["p_l"],
+            "cats_from": ["n_c:0"],
+            "mutual": true,
+            "values": ["romantic", "platonic"],
+            "amount": 46
+},	
+{
+        "cats_to": ["p_l"],
+        "cats_from": ["n_c:0"],
+        "mutual": true,
+        "values": ["trust", "comfort"],
+        "amount": 28
+},	
+{
+        "cats_to": ["p_l"],
+        "cats_from": [ "clan" ],
+        "mutual": false,
+        "values": ["trust"],
+        "amount": -5
+}
+],
+"new_cat": [
+    ["meeting", "clancat", "mate:p_l", "age:mate"]
+],
+"art": "gen_flower_date_warrior_other_clan"
+}
+	],
+	"fail_outcomes": [
+		
+{
+    "text": "{PRONOUN/p_l/subject/CAP}{VERB/p_l/'ve/'s} made up {PRONOUN/p_l/poss} mind. p_l presses {PRONOUN/p_l/poss} pelt to n_c:0's, purring bittersweetly. {PRONOUN/p_l/subject/CAP} {VERB/p_l/have/has} said {PRONOUN/p_l/poss} goodbyes, and so has n_c:0. Now all that's left is to travel to the outer border together to start their shared life.",
+    "exp": 5,
+    "weight": 30,
+    "lost_cats": [ "p_l" ],
+    "other_clan_rep": -3,
+    "relationships": [
+{
+            "cats_to": ["p_l"],
+            "cats_from": ["n_c:0"],
+            "mutual": true,
+            "values": ["romantic", "platonic"],
+            "amount": 46
+},	
+{
+        "cats_to": ["p_l"],
+        "cats_from": ["n_c:0"],
+        "mutual": true,
+        "values": ["trust", "comfort"],
+        "amount": 28
+},	
+{
+        "cats_to": ["p_l"],
+        "cats_from": [ "clan" ],
+        "mutual": false,
+        "values": ["trust"],
+        "amount": -15
+}
+],
+"new_cat": [
+    ["meeting", "clancat", "mate:p_l", "age:mate"]
+],
+"art": "gen_flower_date_warrior_other_clan"
+},
+{
+    "text": "p_l and n_c:0 spend one last night together - and though p_l tries to convince {PRONOUN/n_c:0/object} that the birdsong is the nightangale and not the lark, n_c:0 knows better. {PRONOUN/n_c:0/subject/CAP} can't bear to live in this fantasy any longer. Though p_l calls after n_c:0, {PRONOUN/n_c:0/subject} {VERB/n_c:0/do/does}n't look back as {PRONOUN/n_c:0/subject} {VERB/n_c:0/flee/flees} into the rosy dawn.",

```

Comment:
small typo, ‘nightangale’ should be ‘nightingale’ (taking away ur shakespeare license /j)

Reply:
naur... don't tell my uni